config/routes.rb

```diff
@@ -1,9 +1,15 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  devise_for :admins, path: 'admins'
+  devise_for :employees, path: 'employees'
+
+  authenticated :admin do
```

Comment:
it should be `namespace` - you should be able to access that path, and login afterwards